app/classifier/tasks/dropdown/index.spec.js

```diff
@@ -81,6 +88,23 @@ describe('DropdownTask:static methods', function () {
 });
 
 describe('DropdownTask', function () {
+  describe('with single select', function () {
+    const annotation = { value: [{ value: 0, option: true }] };
+
+    const wrapper = mount(
+      <DropdownTask
+        task={singleSelect}
+        translation={singleSelect}
+        annotation={annotation}
+        onChange={function (a) { return a; }}
+      />, mockReduxStore
+    );
+
+    it('should show that 0 is selected', function () {
+      assert.deepEqual(wrapper.find('#numbers123').find(Select).prop('value'), { label: '0', value: 0 });
```

Comment:
```suggestion
      assert.deepEqual(wrapper.find(Select).prop('value'), { label: '0', value: 0 });
```
I think you can make this change and switch to shallow rendering.